component/Cards/styles.css

```diff
@@ -65,18 +52,13 @@
     position: relative;
     z-index: 1;
```

Comment:
suggestion: can use css variables for z-indexs.

Reply:
Thank you. Will keep in mind